src/backend/decorators/property/property-options.interface.ts

```diff
@@ -110,4 +110,10 @@ export default interface PropertyOptions {
    * @new In version 3.3
    */
   reference?: string;
+
+  /**
+   * Description of field. Shown as hoverable hint after label.
+   * @new In version 5.3
```

Comment:
This will be v5.6

Reply:
You can also remove `@new in version 3.3` from `hideLabel` since it's already old

Reply:
Changed to 5.6 and removed from label